Flexbox/styles.css

```diff
@@ -9,30 +10,64 @@ body {
     text-align: center;
     border: 1px solid #14213D;
     border-radius: 15px 15px 0 0;
-    height: 100px;
     margin: 10px;
     padding: 10px;
     background-color: #14213D;
     color: #FFFFFF;
 }
 
-.form-group>div {
+.form-group {
+    display: flex;
```

Comment:
This is nice! Flex is recommended approach :+1:

Reply:
A ja się akurat nie zgodzę, bo akurat tutaj flex jest bez sensu użyty (zobacz sobie @madpirat w narzędziach developerskich, co się stanie jak wywalisz stąd `display: flex; flex-direction: column;` - spoiler alert: nic). Element form-group oplata tylko jeden element (diva z klasą input-1 lub input-2), więc nie za bardzo ma się czym wykazać ten flex tutaj :) Display: flex ma sens, jeżeli element ma więcej niż jedno dziecko